### PR TITLE
Refactor Build Button

### LINKED ISSUE #86: Change "All" to act like "Resume All"
I got stuck in a situation where "Resume" was just rerunning a single build.  I also have accidentally hit "All" only to clear all of my loadouts.  Both would be solved if "All" was just a "Resume All".  The change isn't a big deal as someone who intended to clear loadouts can stop, do so, and resume.  I also think it logical that build buttons do not affect loadouts (except by completing builds).

### PR DESCRIPTION
**Pull request by claytondaley**
This resolves #86 by:

1. No longer destroying loadouts on "All"
2. Defaulting to (Resume) All
3. Adding an explicit All to the dropdown

It also includes some refactor/cleanup of methods and UI names.

I would normally get buy-in first but this was a nice way to practice UI changes even if you reject the PR (but I hope you don't).

### CODE REVIEW

File: RuneApp/Program.cs [hunk 1]
```diff
@@ -659,9 +659,14 @@ public static class Program {
 			});
 		}
 
+		public static bool HasActiveBuild()
```
Comment (Skibisky) [inline RuneApp/Program.cs:662]: Consider using a get-only property here

File: RuneApp/Main.cs [hunk 2]
```diff
@@ -1033,19 +1038,22 @@ public partial class Main : Form {
...
-			tsBtnBuildsRunOne.DropDown.Close();
+			tsBtnBuildsRun.DropDown.Close();
 		}
 
 		private void tsBtnBuildsRunUpTo_Click(object sender, EventArgs e) {
-			int selected = -1;
 			if (buildList.SelectedItems.Count > 0)
-				selected = (buildList.SelectedItems[0].Tag as Build).Priority;
-			Program.RunBuilds(true, selected);
+            {
```
Comment (Skibisky) [inline RuneApp/Main.cs:1046]: What is your IDE/git config for whitespace?

Comment (Skibisky) [reply RuneApp/Main.cs:1046]: Or maybe this was during my Tab phase and you are using 4 spaces 👀

**Comment by Skibisky** [comment]
Otherwise looks nice. 
Helpful messages instead of just doing nothing :)

**Comment by claytondaley** [comment]
Thanks.  Not all that familiar with C# patterns but I changed it to a property.  I also ran into some build stop issues while I was testing so all commands will now 

1. stop active build or
2. stop time delayed build or 
3. start the indicated build.

File: RuneApp/Main.cs (hunk 2 above)
Comment (claytondaley) [reply RuneApp/Main.cs]: I'm using whatever VS Community defaults to.

**Comment by claytondaley** [comment]
P.S. I'd like to refactor `Main.cs` to get similar methods (like "BuildX") together for easier refactoring and maintenance, but merging changes like that is virtually impossible so you'd need to "freeze" a build that I can reorganize.

**Comment by claytondaley** [comment]
Oh and I dropped in a message that blocks reloads while builds are running as a quick-and-dirty workaround while we get the sync figured out.  We actually need to have both an `isBuilding` and `isLoading` so they can mutually block.

**Comment by claytondaley** [comment]
I'm confused by some of these changes.  Almost like VS is automatically doing code cleanup.  I'm also very confused by the loss of the "Skip" image in that one file.  I didn't try to do anything like that.

**Comment by claytondaley** [comment]
Does this look right for you? For some reason it looks fine in VS but when I run a build, everything is too large.

**Comment by Skibisky** [comment]
I'll check out your branch and see what happens when I look at it

**Comment by claytondaley** [comment]
I figured out part of the problem on my end.  My dev machine uses a 4k monitor with text at 125% which makes VS go crazy and use the 125% shifted sizes and points.  It also insists on changing tabs to spaces which is causing merge issues.

**Comment by Skibisky** [comment]
Ah, I have seen the light and use spaces these days :)
Related to #88, I might try and merge as much of your stuff in, and then reformat all the files >.>

**Comment by claytondaley** [comment]
Let me regenerate both.  I should be able to make everything merge friendly relatively easily.

**Comment by claytondaley** [comment]
If you merge the whitespace commit, this second commit is the "just delta" though it looks like VS 2019 Community is enforcing some kind of best practice on the toolbar icons.

**Comment by Skibisky** [comment]
Yeah, the auto generated stuff ruins everything

**Pull request closed without merging**